EIPS/eip-draft-smartcontract-fee-share.md

```diff
@@ -0,0 +1,18 @@
+---
+eip: 3279
+title: Smart Contract Deployer Transactions Fee Share
+author: tarekskr (@tarekskr)
+status: Draft
```

Comment:
```suggestion
discussions-to: <URL>
status: Draft
```
`discussions-to` is required.  Recommend creating a thread on https://ethereum-magicians.org/ and putting it here.